test/sqlite/sqllogic_command.hpp

```diff
@@ -146,7 +146,7 @@ class SleepCommand : public Command {
 class UnzipCommand : public Command {
 public:
 	// 1 MB
```

Comment:
This comment is now out of date I assume?

Reply:
The comment wasn't correct before this change; The`BUFFER_SIZE` was initialized with 8MB instead of 1MB.